webpack/assets/javascripts/compute_resource/ovirt.js

```diff
@@ -154,6 +153,17 @@ function disableElement(element) {
   element.attr('disabled', 'disabled');
 }
 
+function changeInputValue(input, newInputValue) {
+  const lastValue = input.value;
+  input.value = newInputValue;
+  const event = new Event('input', { bubbles: true });
+  const tracker = input._valueTracker;
+  if (tracker) {
+    tracker.setValue(lastValue);
+  }
```

Comment:
This manipulates the actual DOM of the react component that uses a virtual DOM.
Instead, we can use the store here for storing the value. with this approach, we can manipulate the virtual DOM and force a re-render.
[Here](http://foreman.surge.sh/?path=/docs/introduction-legacy-js--page) you can find documentation for dispatching actions and observing the store for changes from legacy js.